app/Observers/AssetObserver.php

```diff
@@ -11,7 +11,7 @@
 class AssetObserver
 {
     /**
-     * Listen to the User created event.
+     * Listen to the User updating event. This fires automatically every time an asset is saved.
```

Comment:
Might want to update this to say when an _existing_ asset is saved. "Asset is saved" might translate for some developers to `$asset->save()` - which can be an `updating` event or `creating` event, but is always a `saving` event.